### DISCUSSION TITLE
[FEATURE REQUEST] Use Seed Fuzzing to make Variation, just like Midjourney or Dalle2

### DISCUSSION THREAD

**Discussion by LiJT** [hlky/stable-diffusion-webui#269] [Feature suggestions]
**Describe the solution you'd like**
I saw this post by XRA#8888 in Stable diffusion discord official channal, he made this awesome tweak to SD so that we can basically make variations of an image (conceptually similar to Dalle2's variation thing) and also blend between 2 seeds.

This is the link:
https://github.com/lstein/stable-diffusion/pull/184

I am not a coder but I find this extremely fascinating, Dev please have a look maybe it can change the game!

**Comment by hlky** [comment]
Would need to ask @xraxra if we can use that code and implement it in the UI.

**Comment by hlky** [comment]
Moving to discussion btw, feature requests are going on there now to keep issues tidier.

**Comment by bakkot** [comment]
Note that this isn't _just_ like Midjourney or Dalle2 - it doesn't work with arbitrary images, only with images that the model generates. It works by slightly changing the noise input to the model from the noise used for the original image.

img2img is still the only thing which works for arbitrary images right now.

**Reply by nullandboid** [reply]
Would it be possible to use Img2Img with a very low noise amount to generate an image with a prompt describing its contents to make it usable for variations?

**Reply by bakkot** [reply]
Yes, that works fine. Just set the strength to a low value. Try it and see!

**Reply by LiJT** [reply]
Really wish to get this working

**Comment by ghost** [comment]
I got a PR setup for it in #307 👀 

feel free to improve the frontend.. I wasn't sure where the best place for the Variation Amount and Variant Seed so I put them in the advanced tab.

**Reply by LiJT** [reply]
> I got a PR setup for it in #307 👀
> 
> feel free to improve the frontend.. I wasn't sure where the best place for the Variation Amount and Variant Seed so I put them in the advanced tab.

WOW awesome dude!!!
